project/AppDependencies.scala

```diff
@@ -5,15 +5,15 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28"     % "7.8.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"             % "3.32.0-play-28",
+    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28"     % "7.13.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"             % "6.3.0-play-28",
     "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"             % "0.73.0",
```

Comment:
Apologies, didn't notice that this could be upgraded. v0.74.0 is the latest

Reply:
Because the version is being duplicated it might be worth doing something like this:
https://github.com/hmrc/penalties/blob/9c072f32d3e04e5b0df0eee22fd53ace4e048cb7/project/AppDependencies.scala#L6
For both hmrc-mongo and bootstrap-frontend

Reply:
Btw, I'm getting all these versions from the catalogue